catalyst_voices/lib/pages/account/creation/create_keychain/create_keychain_dialog.dart

```diff
@@ -0,0 +1,96 @@
+import 'package:catalyst_voices/pages/account/creation/create_keychain/create_keychain_controller.dart';
+import 'package:catalyst_voices/pages/account/creation/create_keychain/create_keychain_stage.dart';
+import 'package:catalyst_voices/pages/account/creation/create_keychain/stage/stages.dart';
+import 'package:catalyst_voices/pages/account/creation/information_panel.dart';
+import 'package:catalyst_voices/pages/account/creation/task_picture.dart';
+import 'package:catalyst_voices/widgets/widgets.dart';
+import 'package:catalyst_voices_localization/catalyst_voices_localization.dart';
+import 'package:flutter/material.dart';
+
+class CreateKeychainDialog extends StatefulWidget {
+  const CreateKeychainDialog._();
+
+  static Future<String?> show(BuildContext context) {
+    return VoicesDialog.show(
+      context: context,
```

Comment:
can you add routeSettings with name? It's used i.e. in sentry to see where the user went. I used this one for link-wallet:

```
routeSettings: const RouteSettings(name: '/register/link-wallet'),
```